.github/workflows/unit-test.yml

```diff
@@ -212,7 +214,6 @@ jobs:
               uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
               with:
                   name: build-assets
-                  path: ./build
```

Comment:
Here we remove the path so that it extracts all the assets correctly instead of naming them.

Before, since there was one directory in the aftifact, naming it `build` was OK. Now that we have a parent directory with `build` and `build-module` it just goes into the working directory.